translator/reader/read_options.cpp

```diff
@@ -416,13 +413,11 @@ void UsdArnoldReadRenderSettings::Read(const UsdPrim &prim, UsdArnoldReaderConte
             else {
                 // we found the same aov name multiple times, we'll need to add the layerName
                 useLayerName = true;
-                // store the list of aov names that were actually duplicated
-                duplicatedAovs.insert(aovName);
             }
             VtValue aovNameValue;
             // read the parameter "driver:parameters:aov:name" that will be needed if we have merged exrs (see #816)
             std::string layerName = (renderVarPrim.GetAttribute(_tokens->aovSettingName).Get(&aovNameValue, time.frame)) ? 
-                VtValueGetString(aovNameValue, &prim) : renderVarPrim.GetPath().GetName();
+                VtValueGetString(aovNameValue, nullptr) : renderVarPrim.GetPath().GetName();
```

Comment:
same as above, I guess we need to pass `renderVarPrim.GetAttribute(_tokens->aovSettingName)` ?